### PR TITLE
Improvement: Danish language translation

### PR DESCRIPTION
**Pull request by marcusarnfast**
## Description
Improvement to "da-dk" language translations

- Missing translations.
- Typos.
- Irrelevant capitalization.
- Fix missing "|" or replacing "·" to "l"

Fixes #

## Type of Change

- [X] Bugfix
- [X] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [ ] All tests are passing locally
- [ ] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated. PR:

### CODE REVIEW

**Comment by paescuj** [comment]
Hey @marcusarnfast, thank you very much for your contribution!
Usually, updates to translations should be applied via Crowdin, see https://docs.directus.io/contributing/translations.html.
Considering that you have made a lot of changes, we'll exceptionally see that we can upload the changes directly from this pull request.

File: app/src/lang/translations/da-DK.yaml [hunk 1]
```diff
@@ -248,90 +254,96 @@ uuid: UUID
...
 primary_key: Primærnøgle
-foreign_key: Fremmed Nøgle
-finish_setup: Afslut Opsætning
+foreign_key: Fremmednøgle
+finish_setup: Afslut opsætning
 dismiss: Afvis
 raw_value: Rå værdi
-copy_raw_value: Kopier Rå Værdi
+raw_editor_placeholder: "{'{'}{'{'}$trigger.payload.example{'}'}{'}'}"
+raw_value: Rå værdi
```
Comment (paescuj) [inline app/src/lang/translations/da-DK.yaml:321]:
```suggestion
```

File: app/src/lang/translations/da-DK.yaml [hunk 2]
```diff
@@ -924,15 +999,17 @@ page_help_settings_presets_item: >-
   **Forudindstilling Detaljer** — En formular til håndtering af bogmærker og standard samlingsindstillinger.
 page_help_settings_webhooks_collection: '**Gennemse Webhooks** — Viser alle webhooks i projektet.'
 page_help_settings_webhooks_item: '**Webhook Detail** — En formular til oprettelse og håndtering af projektwebhooks.'
+page_help_settings_flows_collection: '**Gennemse flows** — Viser alle flows i projektet.'
+page_help_settings_flows_item:'**Flowdetalje** — Arbejdsområde til styring af en eller flere operationer.'
```
Comment (paescuj) [inline app/src/lang/translations/da-DK.yaml:1003]:
```suggestion
page_help_settings_flows_item: '**Flowdetalje** — Arbejdsområde til styring af en eller flere operationer.'
```

File: app/src/lang/translations/da-DK.yaml [hunk 3]
```diff
@@ -1770,45 +1972,158 @@ panels:
...
+    response_body: Svar organ
+    response_body_all: All Data
+    response_body_last: Data of Last Operation
+  event:
+    name: Begivenhedskrog
+    description: Udløsere på platform eller datahændelser
+    action: 'Handling (ikke-blokerende)'
+    filter: 'Filter (blokerende)'
   webhook:
+     name: Webkrog
```
Comment (paescuj) [inline app/src/lang/translations/da-DK.yaml:2027]:
```suggestion
    name: Webkrog
```

File: app/src/lang/translations/da-DK.yaml [hunk 4]
```diff
@@ -1770,45 +1972,158 @@ panels:
...
+    description: Udløses på en indgående HTTP-anmodning
     method: Metode
+    async: Asynkron
+  operation:
+    name: Andet Flow
+    description: Udløst af et andet flow, til kæde
+    preview:Udløst af en operation i et andet flow
```
Comment (paescuj) [inline app/src/lang/translations/da-DK.yaml:2034]:
```suggestion
    preview: Udløst af en operation i et andet flow
```

File: app/src/lang/translations/da-DK.yaml [hunk 5]
```diff
@@ -1770,45 +1972,158 @@ panels:
...
+    collection_only: Kun samlingsside
+    item_only: Kun element side
+    collection_page: Samlingsside
+    require_selection: Kræver valg
+a_flow_uuid: Vælg et flow, der skal udløses...
+any_string_or_json: Enhver streng eller JSON...
+item_payload_placeholder: Dette er den JSON, der bruges til at opdatere elementets feltværdier...
+operation_variables_note: Du kan få adgang til flowdata med variabler såsom **{'{'}{'{'}$last{'}'}{'}'}**, **{'{'}{'{'}$trigger{'}'}{'}'}** og **{'{'}{'{'}$accountability{'}'}{'}'}**, eller referer til tidligere handlinger ved hjælp af **{'{'}{'{'}operation_key{'}'}{'}'}**.'
 operations:
+condition:
```
Comment (paescuj) [inline app/src/lang/translations/da-DK.yaml:2052]:
```suggestion
  condition:
```

File: app/src/lang/translations/da-DK.yaml [hunk 6]
```diff
@@ -1770,45 +1972,158 @@ panels:
...
     message: Besked
+    message_placeholder: Indtast en besked, der skal vises i konsollen...
+  mail:
+    name: Send e-mail
+    description: Send en mail til en eller flere personer
+    to: Til
+    to_placeholder: Tilføj e-mail-adresser og tryk på enter...
+    body: Body
   notification:
+     name: Send meddelelse
```
Comment (paescuj) [inline app/src/lang/translations/da-DK.yaml:2095]:
```suggestion
    name: Send meddelelse
```

**Comment by marcusarnfast** [comment]
Ohh, that's my bad! Couldn't find it.
I will ofcourse use crowdin next time.
But thanks for merging it directy🙌🏼

**Comment by rijkvanzanten** [comment]
@marcusarnfast Thanks! Crowdin sometimes gets a little confused in syncing if the change is made on GitHub rather than on Crowdin, so I'll upload the changes there as well to make sure things don't automatically disappear again 👍🏻

**Pull request merged**